docs/components/checkboxes/README.md

```diff
@@ -229,7 +229,7 @@ Different sizing for checkboxes.
 
 ## Indeterminate
 
-Displays status for checkbox group by indicating that some of the sub-selections in a list are selected. Provides user with ability to select or unselect all items in the list’s sub-group.
+Displays status for checkbox group by indicating that some of the sub-selections in a list are selected. Provides user with ability to select or unselect all items in the list’s sub-group. To see a functioning example see this [codesandbox](https://codesandbox.io/s/cedar-example-sandbox-rubkk).
```

Comment:
can we move this to the "team sandboxes" in codesandbox and give it a custom url slug? that way any one off sandboxes we might have to update for a release will be in the same place

 I think all of the existing team sandboxes we have in there are out of date and can be deleted, but we can maybe confirm that at the next critique

Reply:
done